packages/core/src/Stores/portfolio-store.js

```diff
@@ -55,7 +55,7 @@ export default class PortfolioStore extends BaseStore {
             barriers: observable,
             main_barrier: observable,
             contract_type: observable,
-            active_positions: observable.shallow,
+            active_positions: observable.struct,
```

Comment:
observers were not updated about internal changes of this observable which resulted in the purchase button for Accumulator being enabled even when a contract was already open while it's supposed to be disabled (only 1 contract is allowed for an asset at a time).